price_feeder/tasks.py

```diff
@@ -510,11 +542,12 @@ def task_price_feed(self, task=None, global_manager=None):
             td_delta.seconds))
 
         # IF is in range or not in range but is in time
-        if re_post_is_in_range or is_in_range or (not is_in_range and is_in_time) or not last_price_oracle_validity:
-
+        is_post = is_in_range or (not is_in_range and is_in_time) or not last_price_oracle_validity
+        is_re_post = re_post_is_in_range
```

Comment:
what's the idea of this two variables? I don't see them having different values in this ctx